### ISSUE TITLE
MD036 considered over-reaching

### ISSUE THREAD

**Issue by philoserf** [markdownlint/markdownlint#88]
I believe MD0036 should not trigger in the following case:

```
# top heading

## heading text

_byline data_

Some body text. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed
do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit
esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.

## new heading text

Some other body text. Lorem ipsum dolor sit amet, consectetur adipisicing
elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit
anim id est laborum.
```

**Comment by mivok** [comment]
Unfortunately, markdownlint has no good way to be able to distinguish between that text and something like:

```
# Section 1

_Subsection 1_

Lorem ipsum dolor sit amet...

_Subsection 2_

Lorem ipsum dolor sit amet...
```

which is the intended use of the rule. My suggestion in your case would be to just disable that rule as you have a valid use for standalone emphasis on its own line.
